addons/packages/contour/1.20.0/test/unittest/contour_test.go

```diff
@@ -0,0 +1,729 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package contour_test
+
+import (
+	"path/filepath"
+	"strings"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/yaml"
+
+	"github.com/vmware-tanzu/community-edition/addons/packages/test/pkg/repo"
+	"github.com/vmware-tanzu/community-edition/addons/packages/test/pkg/ytt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Contour Ytt Templates", func() {
+	var (
+		values string
+		output string
+		err    error
+
+		configDir = filepath.Join(repo.RootDir(), "addons/packages/contour/1.20.0/bundle/config")
+	)
```

Comment:
Maybe this could be a per-case `testData` struct instead to avoid bugs in the test runner.

Reply:
I think these need to stay as-is because of how Ginkgo works, with references to these closure variables in individual specs as well as BeforeEach, JustBeforeEach, etc. It'll handle parallelization properly if we ever need to go that route (see https://onsi.github.io/ginkgo/#spec-parallelization). You can run `ginkgo -p unittest/` to try it out.

Reply:
Oh thanks!
`-p` does make a 2x difference on my little laptop
Should we add it the Makefile target, so it always run it that way?

Reply:
Sure, doesn't hurt.